Adds documentation for transferring a node or delegator between Staking Collections (#481)
onflow/flow: docs/content/staking/staking-collection-staking-delegation-guide.mdx

### Close an active Delegator

The owner of a Staking Collection can use the **Close Stake** ([SL.10](/core-contracts/staking-collection))
transaction with the following arguments:

| Argument                | Type               | Description |
|-------------------------|--------------------|-------------|
| **nodeID**              | `String`           | The nodeID of the node delegating to.  |
| **delegatorID**         | `Optional(UInt32)` | The delegatorID of the delegator to close. |

## Transfer a Node

> A user may transfer an existing Node to another another accounts Staking Collection.

Reviewer: ```suggestion
A user may transfer an existing Node to another another account's Staking Collection.
```

The account to transfer the Node to must have a valid Staking Collection set up.

> Transferring a Node will remove it from the authorizers Staking Collection and deposit it to the receivers Staking Collection.

Reviewer: ```suggestion
Transferring a Node will remove it from the authorizer's Staking Collection and deposit it to the receiver's Staking Collection.
```

> _Note: Once a Node or Delegator has been transferred, it cannot be accessed again by the sender. As well, all staked tokens will be considered staked by the receivers Staking Collection._

Reviewer: ```suggestion
_Note: Once a Node or Delegator has been transferred, it cannot be accessed again by the sender. As well, all staked tokens will be considered staked by the receiver's Staking Collection._
```

<Callout type="warning">
Transferring a Node will result in loss of custody of any Staked tokens for the sender.
</Callout>

The owner of a Staking Collection can use the **Transfer Node** ([SL.11](/core-contracts/staking-collection))
transaction with the following arguments:

| Argument                | Type               | Description |
|-------------------------|--------------------|-------------|
| **nodeID**              | `String`           | The nodeID of the node to transfer.  |
| **to**                  | `Address`          | The address of the account which contains the Staking Collection to transfer the Node to. |

## Transfer a Delegator

> A user may transfer an existing Delegator to another another accounts Staking Collection.

Reviewer: ```suggestion
A user may transfer an existing Delegator to another another account's Staking Collection.
```

The account to transfer the Delegator to must have a valid Staking Collection set up.

> Transferring a Delegator will remove it from the authorizers Staking Collection and deposit it to the receivers Staking Collection.

Reviewer: ```suggestion
Transferring a Delegator will remove it from the authorizer's Staking Collection and deposit it to the receiver's Staking Collection.
```

> _Note: Once a Node or Delegator has been transferred, it cannot be accessed again by the sender. As well, all staked tokens will be considered staked by the receivers Staking Collection._

Reviewer: ```suggestion
_Note: Once a Node or Delegator has been transferred, it cannot be accessed again by the sender. As well, all staked tokens will be considered staked by the receiver's Staking Collection._
```
